curriculum/challenges/english/07-scientific-computing-with-python/learn-classes-and-objects-by-building-a-sudoku-solver/6606c14182435d4bab0de2ee.md

```diff
@@ -0,0 +1,65 @@
+---
+id: 6606c14182435d4bab0de2ee
+title: Step 30
+challengeType: 20
+dashedName: step-30
+---
+
+# --description--
+
+Next, you're going to create a method that checks if a number can be inserted in a specified column of the sudoku board by checking if the number is not already present in that column.
```

Comment:
```suggestion
Next, you're going to create a method that checks if a number can be inserted in a specified column of the Sudoku board by checking if the number is not already present in that column.
```

Reply:
Not sure, if Sudoku should be capitalised. But it is the name of the puzzle game!